04 Implementation/src/beregnKKO/BeregnKKOImpl.java

```diff
@@ -0,0 +1,78 @@
+package beregnKKO;
+
+import entities.*;
+import entities.exceptions.ForaeldereksistererikkeException;
+import entities.exceptions.NavnEksistererException;
+import entities.exceptions.NegativBeloebException;
+
+import java.util.ArrayList;
+
+public class BeregnKKOImpl implements BeregnKKO, Observable {
+    ObserverManager observerManager;
+    KKO kko;
+
+
+    public void angivKKO(String navn, double beloeb, String foraeldersNavn) throws NegativBeloebException, NavnEksistererException, ForaeldereksistererikkeException{
+        if (beloeb < 0) {
+            throw new NegativBeloebException("Beløbet må ikke være negativt");
+        }
+
+        if (kko.navnEksisterer(navn)) {
+            throw new NavnEksistererException();
+        }
+
+        if (!kko.foraelderEksisterer(foraeldersNavn)) {
+            throw new ForaeldereksistererikkeException();
```

Comment:
```suggestion
            throw new ForaelderEksistererIkkeException();
```
fix i forhold til navnekonvention.